ci/release-image/entrypoint.sh

```diff
@@ -5,7 +5,7 @@ set -eu
 # Otherwise the current container UID may not exist in the passwd database.
 eval "$(fixuid -q)"
 
-if [ "${DOCKER_USER-}" ]; then
+if [ "${DOCKER_USER-}" != "${USER-}" ]; then
```

Comment:
I think we'll need to check if `DOCKER_USER` otherwise this branch will run if it's empty.

Actually...we used to have it this way, right? Yeah: https://github.com/cdr/code-server/commit/79443c14ff41f38d3f05e0a53e4aa592e3c0e418

Any idea why we changed it? I feel like there was a reason but I might need to test it to remember exactly why.

Reply:
omfg haha yess!! Yea so that commit linked looks like I just made a dumb mistake. I removed that branch thinking it was simpler but i was wrong.